_drafts/contribute-to-ansible.markdown

```diff
@@ -218,9 +218,26 @@ $ pip install -r ./test/runner/requirements/sanity.txt
     e. push on your github repository
 
 ## Step 3 - Make a pull request
-    a . create PR in github
+
+Congrats ! You should be ready now to create your first pull request (PR) on [ansible project](https://github.com/ansible/ansible).
+
+    a . Create a pull request on ansible github project.
+You now have pushed your new branch into your fork. The next step is now to create the PR. to do this, let's go to the ansible github
+ project page : [https://github.com/ansible/ansible](https://github.com/ansible/ansible). 
+You should see a button 'Compare & pull request' on the top of the page :
+![Compare & pull request](../_img/contribute-to-ansible/create_merge_request.png)
```

Comment:
Wrong jekyll syntaxe for including images, [read the doc](https://jekyllrb.com/docs/posts/#including-images-and-resources)